<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyMusic</class>
 <widget class="QMainWindow" name="MyMusic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1277</width>
    <height>837</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyMusic</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="lay_bg">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget_bg" native="true">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="title_widget" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>67</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>67</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLineEdit" name="find_lineEdit">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>35</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="find_btn">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="icon.qrc">
              <normaloff>:/icon/find.png</normaloff>:/icon/find.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="min_btn">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="icon.qrc">
              <normaloff>:/icon/min.png</normaloff>:/icon/min.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="max_btn">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="icon.qrc">
              <normaloff>:/icon/max.png</normaloff>:/icon/max.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="close_btn">
            <property name="styleSheet">
             <string notr="true">QToolButton:hover {
background-image: url(:/icons/close_hover.png);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="icon.qrc">
              <normaloff>:/icon/close.png</normaloff>:/icon/close.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="2,9">
         <item>
          <widget class="QWidget" name="widget" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QListWidget" name="listWidget">
              <item>
               <property name="text">
                <string>发现音乐</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>博客</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>视频</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>朋友</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>直播</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>私人FM</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>我的音乐</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="myMusicList">
              <item>
               <property name="text">
                <string>本地与下载</string>
               </property>
               <property name="icon">
                <iconset resource="icon.qrc">
                 <normaloff>:/icon/下载.png</normaloff>:/icon/下载.png</iconset>
               </property>
              </item>
              <item>
               <property name="text">
                <string>最近播放</string>
               </property>
               <property name="icon">
                <iconset resource="icon.qrc">
                 <normaloff>:/icon/最近播放.png</normaloff>:/icon/最近播放.png</iconset>
               </property>
              </item>
              <item>
               <property name="text">
                <string>我的音乐云盘</string>
               </property>
               <property name="icon">
                <iconset resource="icon.qrc">
                 <normaloff>:/icon/云盘.png</normaloff>:/icon/云盘.png</iconset>
               </property>
              </item>
              <item>
               <property name="text">
                <string>我的博客</string>
               </property>
               <property name="icon">
                <iconset resource="icon.qrc">
                 <normaloff>:/icon/播客.png</normaloff>:/icon/播客.png</iconset>
               </property>
              </item>
              <item>
               <property name="text">
                <string>我的收藏</string>
               </property>
               <property name="icon">
                <iconset resource="icon.qrc">
                 <normaloff>:/icon/收藏.png</normaloff>:/icon/收藏.png</iconset>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QStackedWidget" name="stackedWidget">
           <property name="currentIndex">
            <number>4</number>
           </property>
           <widget class="QWidget" name="page">
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QTabWidget" name="tabWidget">
               <property name="currentIndex">
                <number>1</number>
               </property>
               <widget class="QWidget" name="tab">
                <attribute name="title">
                 <string>个性推荐</string>
                </attribute>
                <widget class="QLabel" name="label">
                 <property name="geometry">
                  <rect>
                   <x>240</x>
                   <y>250</y>
                   <width>72</width>
                   <height>15</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>TODO</string>
                 </property>
                </widget>
               </widget>
               <widget class="QWidget" name="tab_2">
                <attribute name="title">
                 <string>专属推荐</string>
                </attribute>
                <widget class="QLabel" name="label_2">
                 <property name="geometry">
                  <rect>
                   <x>250</x>
                   <y>260</y>
                   <width>72</width>
                   <height>15</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>TODO</string>
                 </property>
                </widget>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="page_2">
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="1" column="0">
              <widget class="MyTable" name="tableWidget"/>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>最近播放</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="page_3">
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="1" column="0">
              <widget class="QListWidget" name="listWidget_3"/>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>我的音乐云盘</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="page_4">
            <widget class="QLabel" name="label_4">
             <property name="geometry">
              <rect>
               <x>220</x>
               <y>140</y>
               <width>211</width>
               <height>51</height>
              </rect>
             </property>
             <property name="text">
              <string>我的博客 TODO</string>
             </property>
            </widget>
           </widget>
           <widget class="QWidget" name="page_5">
            <widget class="QLabel" name="label_5">
             <property name="geometry">
              <rect>
               <x>270</x>
               <y>210</y>
               <width>191</width>
               <height>31</height>
              </rect>
             </property>
             <property name="text">
              <string>我的收藏 TODO</string>
             </property>
            </widget>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QWidget" name="tail_widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QToolButton" name="image">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <property name="spacing">
                 <number>20</number>
                </property>
                <item>
                 <widget class="QToolButton" name="toolButton">
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="icon.qrc">
                    <normaloff>:/icon/上一曲1.png</normaloff>:/icon/上一曲1.png</iconset>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="play">
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="icon.qrc">
                    <normaloff>:/icon/play.png</normaloff>:/icon/play.png</iconset>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="toolButton_2">
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="icon.qrc">
                    <normaloff>:/icon/下一曲1.png</normaloff>:/icon/下一曲1.png</iconset>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="current_time">
                <property name="text">
                 <string>0:00</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="state">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="end_time">
                <property name="text">
                 <string>0:00</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyTable</class>
   <extends>QTableWidget</extends>
   <header location="global">mytable.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
